### ISSUE TITLE
In power point 2016, NVDA repeats very often the same tool bar or group title when navigating the ribbons by tab and shift+tab

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8022]
### Steps to reproduce:
1. Open Power Point 2016
2. Press alt and then right arrow to navigate to the "insert" ribbon
3. press tab and shift+tab to navigate through the items

### Expected behavior:
NVDA should report a group title only once I have entered that group (i.e. illustration).

### Actual behavior:
NVDA repeats very often the same group title or tool bar title after I have entered the group with tab or shift+tab.

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Both

Other information:

Windows version:
Windows 10.16299.64

Name and version of other software in use when reproducing the issue:
MS Power Point 2016

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by leonardder** [comment]
Could you make sure that you have the most recent version of Office 2016? Also, does this problem occur for you in Excel/Word/Outlook as well?

**Comment by Adriani90** [comment]
Hello,

 

yes this is the very last version.

I can reproduce this issue only in Power Point.

 

 

Von: Leonard de Ruijter [mailto:notifications@github.com] 
Gesendet: Mittwoch, 21. Februar 2018 20:53
An: nvaccess/nvda <nvda@noreply.github.com>
Cc: Adriani90 <adriani.botez@googlemail.com>; Author <author@noreply.github.com>
Betreff: Re: [nvaccess/nvda] In power point 2016, NVDA repeats very often the same tool bar or group title when navigating the ribbons by tab and shift+tab (#8022)

 

Could you make sure that you have the most recent version of Office 2016? Also, does this problem occur for you in Excel/Word/Outlook as well?

**Comment by Adriani90** [comment]
I will close this because it seems that this issue is happening in MS Office in general. I have the same issue in MS Word and Excel. I have created a new issue, see
#8391
